docs/REFERENCE.md

```diff
@@ -2116,6 +2117,57 @@ accounts).
 
 ---
 
+### `onepasswordItemFields` *uuid* [*vault-uuid* [*account-name*]]
+
+`onepasswordItemFields` returns structured data from
+[1Password](https://1password.com/) using the [1Password
+CLI](https://support.1password.com/command-line-getting-started/) (`op`). *uuid*
+is passed to `op get item <uuid>`, the output from `op` is parsed as JSON, and
+each element of `details.sections` are iterated over and any `fields` are
+returned as a map indexed by each field's `n`. For example, give the output from
+`op`:
+
+```json
+{
+  "uuid": "<uuid>",
+  "details": {
+    "sections": [
+      {
+        "name": "linked items",
+        "title": "Related Items"
+      },
+      {
+        "fields": [
+          {
+            "k": "string",
+            "n": "D4328E0846D2461E8E455D7A07B93397",
+            "t": "exampleLabel",
+            "v": "exampleValue"
+          }
+        ],
+        "name": "Section_20E0BD380789477D8904F830BFE8A121",
+        "title": ""
+      }
+    ]
+  },
+}
+```
+
+the return value will be the map:
+
+```json
+{
+    "exampleLabel": {
+        "k": "string",
+        "n": "D4328E0846D2461E8E455D7A07B93397",
+        "t": "exampleLabel",
+        "v": "exampleValue"
+    }
+}
+```
+
+---
+
```

Comment:
I think the readme should provide the equivalent commands

```shell
op get item abcdefghijklmnopqrstuvwxyz --fields "exampleLabel"
exampleValue
```

```go-template
{{ (onepasswordItemFields "abcdefghijklmnopqrstuvwxyz").exampleLabel.v }}
```

Reply:
Added in a79ba2f0699ddbc1245715da12975daec18db513.